recipes/occt/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: occt
+  version: 7.0.0
+
+source:
+  fn: occt.tar.gz
+  url: http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=1d505bb0f8a64feb87c76a230b64168242461d94;sf=tgz
+
+build:
+  number: 0
+
+requirements:
+  build:
+  #   track_features:
```

Comment:
Yup, so here you just put the `vc14` feature for Python 3.5. And then you need to include Python as a build dependency to ensure that the features are activated for building. You will also put the `win` selector on the `python` dep.